gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -816,15 +815,14 @@ public static void validateCopyArguments(
       List<String> dstObjectNames,
       GoogleCloudStorage gcsImpl)
       throws IOException {
-    Preconditions.checkArgument(!Strings.isNullOrEmpty(srcBucketName),
-        "srcBucketName must not be null or empty");
-    Preconditions.checkArgument(!Strings.isNullOrEmpty(dstBucketName),
-        "dstBucketName must not be null or empty");
-    Preconditions.checkArgument(srcObjectNames != null,
-        "srcObjectNames must not be null");
-    Preconditions.checkArgument(dstObjectNames != null,
-        "dstObjectNames must not be null");
-    Preconditions.checkArgument(srcObjectNames.size() == dstObjectNames.size(),
+    Preconditions.checkArgument(
```

Comment:
Revert formatting changes? (here and below)

Reply:
Can we confirm that we use google-java-format to format across the board?

This format is what i got after I switch to google-java-format and I dont know how to format back to original version..